apps/instance_app/main.tf

```diff
@@ -0,0 +1,22 @@
+module "child" {
+  source = "../../modules/ec2-eni"
+
```

Comment:
de ce nu se transmite tipul instantei, ami, ip-ul pentru eni ?

si de ce nu e nimic in variables.tf?